db/migrate/20161102045612_add_resolve_url_option_to_website_agent.rb

```diff
@@ -0,0 +1,19 @@
+class AddResolveUrlOptionToWebsiteAgent < ActiveRecord::Migration[5.0]
+  def up
+    Agents::WebsiteAgent.find_each do |agent|
+      keys = agent.event_keys
```

Comment:
I added it in a recent commit for use in `event_description` to fix a bug introduced when adding the `template` option.  Sorry for confusing you!